packages/cli/src/zosuss/ZosUss.definition.ts

```diff
@@ -14,8 +14,8 @@ import { IssueCommand } from "./issue/Issue.definition";
 import { SshSession } from "@zowe/zos-uss-for-zowe-sdk";
 
 export const definition: ICommandDefinition = {
-    name: "zos-uss",
-    aliases: ["uss"],
+    name: "zos-ssh",
+    aliases: ["ssh", "zowe-uss", "uss"],
```

Comment:
Changes look good to me.
There might be a few snapshots that need updated.
Feel free to run `npm run test:unit -- -u` in order to update any snapshots 😋

Reply:
The alias 'zowe-uss' needs to be 'zos-uss' or else this change will be breaking.